gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -490,22 +494,46 @@ def assign_pops(
     :param max_mislabeled_training_samples: Maximum number of training samples that can be mislabelled
     :param n_pcs: Number of PCs to use in the RF
     :param withhold_prop: Proportion of training pop samples to withhold from training will keep all samples if `None`
-    :return: Table of pop assignments and the rf model
+    :param subpops: True if subpops should be assigned rather than pops
+    :param subpops_pca_scores: Table with PCA scores for subpops, needed if `subpops` parameter is True
+    :return: Table of pop or subpop assignments and the rf model
     :rtype: hl.Table
     """
     logger.info("Assigning global population labels")
-    pop_pca_scores_ht = ancestry_pca_scores(include_unreleasable_samples).ht()
-    project_meta_ht = project_meta.ht()[pop_pca_scores_ht.key]
-    pop_pca_scores_ht = pop_pca_scores_ht.annotate(
-        training_pop=(
-            hl.case()
-            .when(
-                hl.is_defined(project_meta_ht.project_pop), project_meta_ht.project_pop
+
+    if subpops:
+        if subpops_pca_scores is None:
+            raise ValueError(
+                "A subpops_pca_scores must be supplied when setting subpops to True"
+            )
+        pop_pca_scores_ht = subpops_pca_scores
+        pop_pca_scores_ht = pop_pca_scores_ht.annotate(
+            training_pop=(
+                hl.case()
+                .when(
+                    hl.is_defined(pop_pca_scores_ht.project_meta.subpop_description),
+                    pop_pca_scores_ht.project_meta.subpop_description,
+                )
+                .or_missing()
             )
-            .when(project_meta_ht.v2_pop != "oth", project_meta_ht.v2_pop)
-            .or_missing()
         )
-    )
+        pop_field = "subpop_description"
```

Comment:
Let's keep it simple and just change to `subpop`
```suggestion
        pop_field = "subpop"
```